src/lightning/data/__init__.py

```diff
@@ -1,12 +1,10 @@
 from lightning.data.datasets import LightningDataset, LightningIterableDataset
-from lightning.data.streaming.dataloader import StreamingDataLoader
 from lightning.data.streaming.dataset import StreamingDataset
-from lightning.data.streaming.dataset_optimizer import DatasetOptimizer
+from lightning.data.streaming.map import map
```

Comment:
please do not use the same name as for building fn
https://www.geeksforgeeks.org/python-map-function/

Reply:
This is coming from @williamFalcon